symfony/plugins/orangehrmLeavePlugin/Service/LeaveAssignmentService.php

```diff
@@ -19,101 +19,120 @@
 
 namespace OrangeHRM\Leave\Service;
 
+use DateTime;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\ORM\EntityManagerHelperTrait;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Entity\Leave;
+use OrangeHRM\Entity\LeaveRequest;
+use OrangeHRM\Entity\LeaveRequestComment;
+use OrangeHRM\Entity\LeaveStatus;
+use OrangeHRM\Entity\WorkflowStateMachine;
 use OrangeHRM\Leave\Dto\LeaveParameterObject;
+use OrangeHRM\Leave\Exception\LeaveAllocationServiceException;
+use OrangeHRM\Leave\Traits\Service\LeaveEntitlementServiceTrait;
 
-class LeaveAssignmentService extends AbstractLeaveAllocationService {
+class LeaveAssignmentService extends AbstractLeaveAllocationService
+{
 
     protected $leaveEntitlementService;
+    use LeaveEntitlementServiceTrait;
+    use EntityManagerHelperTrait;
+    use AuthUserTrait;
+
     protected $dispatcher;
-    protected $assignWorkflowItem;
+    protected $assignWorkflowItem = null;
 
     /**
      * Get LeaveEntitlementService
      * @return LeaveEntitlementService
-     * 
+     *
      */
-    public function getLeaveEntitlementService() {
+    public function getLeaveEntitlementService()
```

Comment:
remove this and see tests are failing. since you used `use LeaveEntitlementServiceTrait;` we don't need to initialize `LeaveEntitlementService` here.

Reply:
done

Reply:
seems not removed

Reply:
i mistakenly changed as to TODO instead of removing.